packages/stream_chat_flutter/CHANGELOG.md

```diff
@@ -46,6 +46,7 @@ breakdown:
 * `UserListViewTheme` is now `UserListViewThemeData`
 
 - Updated core dependency.
+- `UserAvatar` no longer uses a clipper by default.
```

Comment:
this is in the wrong place
I think we don't really need to add an entry for this